src/utils/licenses.ts

```diff
@@ -35,11 +35,16 @@ const licenseEndpoints = [
 async function fetchLicensesForEndpoint(
   url: string,
   apiKey: string,
-  mapperFunc: (license: AnalyticsLicense) => SelectableValue
+  mapperFunc: (license: AnalyticsLicense) => SelectableValue,
+  tenantOrgId?: string
 ) {
+  const headers = { 'X-Api-Key': apiKey, 'X-Tenant-Org-Id': tenantOrgId}
```

Comment:
I think something like this would be a nicer approach 🙂 

```
const headers: Record<string, string> = { 'X-Api-Key': apiKey };

if (tenantOrgId != null) {
  headers['X-Tenant-Org-Id'] = tenantOrgId;
}
```